lamassu-remote-install/install

```diff
@@ -0,0 +1,257 @@
+SERVER_CERT_PATH=$CERT_DIR/Lamassu_OP.pem
+SEEDS_DIR=$HOME/seeds
+SEED_FILE=$SEEDS_DIR/seed.txt
+BACKUP_DIR=/var/backups/postgresql
+BLOCKCHAIN_DIR=/mnt/blockchains
+OFAC_DATA_DIR=/var/lamassu/ofac
+
+# Look into http://unix.stackexchange.com/questions/140734/configure-localtime-dpkg-reconfigure-tzdata
+
+decho () {
+  echo `date +"%H:%M:%S"` $1
+  echo `date +"%H:%M:%S"` $1 >> $LOG_FILE
+}
+
+retry() {
+  local -r -i max_attempts="$1"; shift
+  local -r cmd="$@"
+  local -i attempt_num=1
+
+  until $cmd
+  do
+    if (( attempt_num == max_attempts ))
+    then
+        echo
+        echo "****************************************************************"
+        echo "Attempt $attempt_num failed and there are no more attempts left! ($cmd)"
+        return 1
+    else
+        echo
+        echo "****************************************************************"
+        echo "Attempt $attempt_num failed! Trying again in $attempt_num seconds..."
+        sleep $(( attempt_num++ ))
+    fi
+  done
+}
+
+rm -f $LOG_FILE
+
+cat <<'FIG'
+ _
+| | __ _ _ __ ___   __ _ ___ ___ _   _       ___  ___ _ ____   _____ _ __
+| |/ _` | '_ ` _ \ / _` / __/ __| | | |_____/ __|/ _ \ '__\ \ / / _ \ '__|
+| | (_| | | | | | | (_| \__ \__ \ |_| |_____\__ \  __/ |   \ V /  __/ |
+|_|\__,_|_| |_| |_|\__,_|___/___/\__,_|     |___/\___|_|    \_/ \___|_|
+FIG
+
+echo -e "\nStarting \033[1mlamassu-server\033[0m install. This will take a few minutes...\n"
+
+if [ "$(whoami)" != "root" ]; then
+  echo -e "This script has to be run as \033[1mroot\033[0m user"
+  exit 3
+fi
+
+release=$(lsb_release -rs)
+processor=$(uname -i)
+if [ "$release" != "16.04" ] || [ "$processor" != "x86_64" ]; then
+    echo "You're attempting to install on an unsupported Linux distribution or release."
+    uname -a
+    echo "Please return to DigitalOcean and create a droplet running Ubuntu 16.04 x64 instead."
+    exit 1
+fi
+
+# So we don't run out of memory
+decho "Enabling swap file for install only..."
+fallocate -l 1G /swapfile >> $LOG_FILE 2>&1
+chmod 600 /swapfile >> $LOG_FILE 2>&1
+mkswap /swapfile >> $LOG_FILE 2>&1
+swapon /swapfile >> $LOG_FILE 2>&1
+
+IP=$(ifconfig eth0 | grep "inet" | grep -v "inet6" | awk -F: '{print $2}' | awk '{print $1}')
+
+decho "Updating system..."
+sleep 10
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - >> $LOG_FILE 2>&1
+apt update >> $LOG_FILE 2>&1
+
+decho "Installing necessary packages..."
+apt install nodejs python-minimal build-essential supervisor postgresql libpq-dev -y -q >> $LOG_FILE 2>&1
+
+decho "Generating seed..."
+mkdir -p $SEEDS_DIR >> $LOG_FILE 2>&1
+SEED=$(openssl rand -hex 32)
+echo $SEED > $SEED_FILE
+
+decho "Installing latest npm package manager for node..."
+retry 3 npm -g --unsafe-perm install npm@5  >> $LOG_FILE 2>&1
+NODE_MODULES=$(npm -g root)
+NPM_BIN=$(npm -g bin)
+
+decho "Installing lamassu-server..."
+retry 3 npm -g --unsafe-perm install lamassu/lamassu-server#master >> $LOG_FILE 2>&1
```

Comment:
hey @joshmh I was thinking about the branch name: what if we have the branch name as parameter?
it can default to `master` otherwise we can pass in something like:
```
curl -sS https://raw.githubusercontent.com/lamassu/lamassu-server/master/lamassu-remote-install/install | bash -s -- master
```

Reply:
Yeah, I like that idea. @DatoBula any reason not to do this?

Reply:
No. I will do this.